frontend/public/components/deployment-config.tsx

```diff
@@ -67,9 +67,38 @@ const menuActions = [
   ...common,
 ];
 
+const ContainerRow = ({container}) => {
+  const resourceLimits = _.get(container, 'resources.limits');
+  const ports = _.get(container, 'ports');
+  return <div className="row">
+    <div className="col-xs-6 col-sm-4 col-md-3">
+      {container.name}
+    </div>
+    <Overflow className="col-xs-6 col-sm-4 col-md-3" value={container.image} />
+    <div className="col-sm-4 col-md-3 hidden-xs">{resourceLimits ? _.map(resourceLimits, (v, k) => `${k}: ${v}`).join(', ') : '-'}</div>
```

Comment:
Maybe just

```jsx
{_.map(resourceLimits, (v, k) => `${k}: ${v}`).join(', ') || '-'}
```

If we keep the ternary, better to check `_.isEmpty(resourceLimits)` to handle empty arrays.